### PR TITLE
[Aidai] Added TypeScript support

### PR DESCRIPTION
**Pull request by networkflow**
Summary:
* Installed TypeScript in the project

Test plan:
* It still works

### CODE REVIEW

File: dist/tic-tac-toe.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+"use strict";
+/* Produces a diagram that looks like this:
```
Comment (sydykbaeva-a) [inline dist/tic-tac-toe.js:2]: Can you clarify "use strict" - is it the comment?

File: dist/tic-tac-toe.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+function getInitialBoard() {
+    // If height is larger than 9, we need to adjust the label widths
+    // to account for two-digit labels:
+    const H = 3;
+    // If the width is larger than 26, we need to adjust the column labels
+    // to account for labels after Z:
+    const W = 3;
+    let boardString = '';
+    // Column labels:
+    for (let j = 0; j < W; j += 1) {
```
Comment (sydykbaeva-a) [inline dist/tic-tac-toe.js:19]: for the iterators maybe rename them - row, column - so it would be more clear

File: dist/tic-tac-toe.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+            if (j === 0) {
+                boardString += (i + 1).toString() + '    ';
+                continue;
+            }
+            else {
+                boardString += '|   ';
+            }
+        }
+        boardString += '\n';
+    }
```
Comment (sydykbaeva-a) [inline dist/tic-tac-toe.js:48]: Overall, the code looks good! Although, could you include the edge cases in Test Plan.
Maybe making values for this: '  ---', '----',  '|   ' in case if we decide to change pattern

**Review by sydykbaeva-a — CHANGES_REQUESTED** [review]
